gdal/swig/perl/Makefile.PL

```diff
@@ -53,7 +53,7 @@ for my $module (sort keys %$objects) {
     my $LD = $Config{ld};
     $LD .= ' '.$ENV{CFLAGS} if $ENV{CFLAGS};
     $LD .= ' '.$ENV{LDFLAGS} if $ENV{LDFLAGS};
-    my $OPTIMIZE = '';
+    my $OPTIMIZE = '-std=c++11';
```

Comment:
@ajolma Would there be a way of dynamically checking if we need to add this flag ? Something in spirit similar to the Python equivalent in https://github.com/OSGeo/gdal/pull/270/commits/e2d6633eed23514b1b61d8fa52b64b1e58f4576d

Reply:
Or perhaps use the CXX variable from GDALmake.opt (which has -std=c++11 if needed)

Reply:
See above, I guess. We can put the switch into CC => 'g++ -std=c++11' (actually in a variable that is picked from the opt.